requirements.txt

```diff
@@ -0,0 +1,4 @@
+chardet<4
```

Comment:
I used `chardet` as an example.

Usually for libraries, you pin major versions of dependencies, since they may break compatibility.

Reply:
During offline talks we decided not to pin upper limits.

You may remove the upper limit also on this requirement.